app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,81 @@
+class ExpensesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_expense, only: %i[show edit update destroy]
+
+  # GET /expenses or /expenses.json
+  def index
+    @group = current_user.groups.find(params[:group_id])
```

Comment:
Please refactor this piece of code when you find a group by group id which is used in almost every action of the controller. One way is to define a private method  and use `before_action` filter as you are using for `#set_expense`

Reply:
Thank you @SajjadAhmad14  for your review, I also mentioned to the previous reviewer that when I try this it's giving me errors, on my expense page.

![image](https://user-images.githubusercontent.com/106079814/204057613-08430d7c-6450-4e8a-a7ef-a78bc6e7d96d.png)